outputs.tf

```diff
@@ -2,7 +2,24 @@
 # Outputs
 ########################################################################################################################
 
-#output "myoutput" {
-#  description = "Description of my output"
-#  value       = "value"
-#}
+output "name" {
+  description = "Name of created SCC WP instance."
+  value       = ibm_resource_instance.scc_wp.name
+}
+
+output "id" {
+  description = "ID of created SCC WP instance."
+  value       = ibm_resource_instance.scc_wp.crn
+}
+
+output "sysdig_collector_endpoint" {
+  description = "Sysdig collector endpoint."
+  value       = length(var.scc_wp_keys) > 0 ? ibm_resource_key.scc_wp_keys[keys(var.scc_wp_keys)[0]].credentials["Sysdig Collector Endpoint"] : null
```

Comment:
this means if the consumer chooses not to create a resource key (credential), this output will not get returned. I took a look at the observability instances module, and it always creates a resouce key. Should we do that here too? Lets chat tomorrow

Reply:
resource key must be created and we always created it inside the module. I used the length checking to prevent a failure in a case, resource keys are not generated (in a case backend service fails). Is it over engineered and we should leave to fail in a case that keys are not generated?